### PR TITLE
Update bits of matomo docs

### PR DESCRIPTION
**Pull request by J0WI**
Just a small update to address the feedback in https://github.com/matomo-org/docker/issues/69

### CODE REVIEW

File: matomo/content.md [hunk 1]
```diff
@@ -58,3 +60,19 @@ We'd love to hear your feedback and suggestions in the issue tracker: [github.co
 ## GeoIP
 
 This product includes GeoLite data created by MaxMind, available from [http://www.maxmind.com](http://www.maxmind.com).
+
+## Image Variants
```
Comment (yosifkit) [inline matomo/content.md]: This section is duplicating the generated section: https://github.com/docker-library/docs/tree/2158db126ad9a21a15c6d1d46ac88b4a09f3966b/matomo#image-variants;  You probably just need separate `variant-apache.md` and `variant-fpm.md files` like [php](https://github.com/docker-library/docs/tree/2158db126ad9a21a15c6d1d46ac88b4a09f3966b/php).

Comment (J0WI) [reply matomo/content.md]: Removed the duplicate.

**Comment by yosifkit** [comment]
LGTM

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**